tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecSuite.scala

```diff
@@ -342,16 +343,22 @@ class AdaptiveQueryExecSuite
       // the read should be an adaptive plan
       val adaptiveSparkPlanExec = TestUtils.findOperator(writeCommand.get,
         _.isInstanceOf[AdaptiveSparkPlanExec])
-          .get.asInstanceOf[AdaptiveSparkPlanExec]
+        .get.asInstanceOf[AdaptiveSparkPlanExec]
 
-      val transition = adaptiveSparkPlanExec
+      if (cmpSparkVersion(3, 2, 0) < 0) {
```

Comment:
This is likely to cause problems with at least the upcoming Databricks 8.3 shim, as that's an example of a shim that is _partially_ based on 3.2 code.  Since it's already shipped, I do not expect it to contain the fix that we're looking for with this check here.

Reply:
I have removed the version check and added a `supportsColumnarAdaptivePlans` method to the shim layer